src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -4,10 +4,11 @@
     <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.5.1</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
```

Comment:
I don't think we want conditions on this.  All the projects added to this repo should be set up to ship initially since we need to ship one version that establishes the new package shape and supported frameworks.

Reply:
While these conditions aren't necessary to ship the package initially, it follows my versioning proposal for subsequent releases. I decided to set `IsPackable` to false until we start shipping these packages. We can flip that right now, if we want to.